config/routes.rb

```diff
@@ -151,6 +151,15 @@
     get '/shares',            controller: 'my/shares', action: :index, as: 'dashboard_shares'
     get '/shares/page/:page', controller: 'my/shares', action: :index
     get '/shares/facet/:id',  controller: 'my/shares', action: :facet, as: 'dashboard_shares_facet'
+    scope :collections do
+      get '/permission_template/new' => 'admin/permission_templates#new', as: :new_dashboard_collection_permission_template
```

Comment:
I ask partly because I was wondering if we could replace these explicit routes with something like:

```ruby
resources :collections, only: [] do
  resource :permission_template
end
```

Or something like that?